disperser/controller/dispatcher.go

```diff
@@ -534,25 +536,28 @@ func (d *Dispatcher) updateBatchStatus(ctx context.Context, batch *batchData, qu
 			if err != nil {
 				multierr = multierror.Append(multierr, fmt.Errorf("failed to update blob status for blob %s to failed: %w", blobKey.Hex(), err))
 			}
+			d.metrics.reportCompletedBlob(int(cert.BlobHeader.BlobCommitments.Length)*encoding.BYTES_PER_SYMBOL, v2.InsufficientSignatures)
 			continue
 		}
 
 		err := d.blobMetadataStore.UpdateBlobStatus(ctx, blobKey, v2.Certified)
 		if err != nil {
 			multierr = multierror.Append(multierr, fmt.Errorf("failed to update blob status for blob %s to certified: %w", blobKey.Hex(), err))
 		}
+		d.metrics.reportCompletedBlob(int(cert.BlobHeader.BlobCommitments.Length)*encoding.BYTES_PER_SYMBOL, v2.Certified)
```

Comment:
Not sure if this is the right method but it gives the nearest power of 2 for throughput.